/**:
  ros__parameters:
    odom_freq: 20
    global_frame_id: "map"
    odom_frame_id: "odom"
    base_frame_id: "base_link"
    footprint_frame_id: "base_footprint"
    laser_min_range: 0.0
    laser_max_range: 100000000.0
    scan_increment: 1
    initial_pose_x: 88.99264526367188
    initial_pose_y: 135.97076416015625
    initial_pose_a: 3.14
    num_particles: 1000
    alpha_threshold: 0.5
    expansion_radius_position: 1.0
    expansion_radius_orientation: 0.2
    extraction_rate: 0.1
    range_threshold: 0.3
    sensor_reset: false
    odom_fw_dev_per_fw: 0.5
    odom_fw_dev_per_rot: 0.2
    odom_rot_dev_per_fw: 0.13
    odom_rot_dev_per_rot: 0.05
    laser_likelihood_max_dist: 0.2
    use_gnss_reset: true
    use_wall_tracking: true
    gnss_reset_var: 9.0
    kld_th: 5.0
    pf_var_th: 1.5
